crates/re_viewer/src/ui/welcome_screen/example_page.rs

```diff
@@ -379,72 +369,109 @@ impl ExamplePage {
             });
         });
 
-        WelcomeScreenResponse::default()
-    }
-}
+        if !examples_visible {
+            let screen_rect = ui.ctx().screen_rect();
+            let indicator_rect = examples_page_rect
+                .with_min_y(screen_rect.bottom() - 125.0)
```

Comment:
I know for a fact that it isn't always convenient to chase every last magic number out, but this one is a little bit scary. Is this going to be robust enough to other changes?

I'm not sure it is relevant here, but I sometime "record" the current screen position after some UI elements by storing `ui.cursor().min.y` in a variable which can then be used later on. Maybe that pattern applies here?